Palovaara, J., Shunsuke, S., Wendrich, J.R., van ’t Wout Hofland, N., van Schayck, P., Hater, F., Mutte, S., 
Sjollerma, J., Boekschoten, M., Hooiveld, G.J., Weijers, D. (2017). Transcriptome dynamics revealed by a gene 
expression atlas of the early Arabidopsis embryo. Nat. Plants, in press.

[https://doi.org/10.1038/s41477-017-0035-3](https://doi.org/10.1038/s41477-017-0035-3)

- Cell type-specific nuclei were isolated from 16-cell stage, early globular stage and late globular stage 
  Arabidopsis embryos using the INTACT method[^Deal and Henikoff 2010].
- Plants were grown under at a constant temperature of 22° under long-day conditions (16 hours light/8 hours dark).
- RNA samples representing up to four biological replicates were extracted and >500 pg of total RNA was amplified and 
  labelled using the Ovation Pico WTA System V2 with the Encore® Biotin Module (NuGEN).
- Labelled samples were hybridized to an Arabidopsis Gene 1.1 ST 24-Array Plate (Affymetrix).
- Microarray analysis was performed using MADMAX pipeline for statistical analysis of microarray data[^Lin et al. 2011]. 
  Expression values (signal intensities) were normalized with RMA normalization.
- Fold changes are Limma log2-based[^Smyth 2002] and differences with q-values < 0.05 are considered significant.

Gene Expression Omnibus (GEO): [GSE89100](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89100)

[^Deal and Henikoff 2010]: Deal RB and Henikoff S. (2010). A simple method for gene expression and chromatin profiling of individual cell types within a tissue. Dev Cell. 18: 1030-1040.

[^Lin et al. 2011]: Lin et al. (2011). MADMAX-Management and analysis database for multiple –omics experiments. J Integr Bioinform. 8(160): doi: 10.2390/biecoll-jib-2011-160; pmid: 21778530.
 
[^Smyth 2002]: Smyth GK. (2004). Linear models and empirical bayes methods for assessing differential expression in microarray experiments. Stat Appl Genet Mol Biol. 3: Article3. 